modules/mobile/app/serializers/mobile/v2/user_serializer.rb

```diff
@@ -22,8 +22,9 @@ def initialize(user)
 
       # this is a temporary fix
       def transitioning_facility?(user)
+        cerner_facility_id = Settings.vsp_environment == 'production' ? '556' : '979'
         Flipper.enabled?(:mobile_cerner_transition, user) &&
-          user.va_treatment_facility_ids.include?('556')
+          user.vha_facility_ids.include?(cerner_facility_id)
```

Comment:
I had to change from va_treatment_facility_ids to vha_facility_ids because va_treatment_facility_ids has to be within a certain numerical range that 979 is not in. Which is why they should not have used 979 as a test id.